e2e/bitrise.yml

```diff
@@ -0,0 +1,296 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # Shared test configs
+  - BITRISE_KEYCHAIN_PATH: $HOME/Library/Keychains/login.keychain
+  # Shared test secrets
+  - BITRISE_KEYCHAIN_PASSWORD: $BITRISE_KEYCHAIN_PASSWORD
+  - BITRISE_CERTIFICATE_URL_LIST: $BITRISE_CERTIFICATE_URL_LIST
+  - BITRISE_CERTIFICATE_PASSPHRASE_LIST: $BITRISE_CERTIFICATE_PASSPHRASE_LIST
+  - BITRISE_PROVISIONING_PROFILE_URL_LIST: $BITRISE_PROVISIONING_PROFILE_URL_LIST
+
+workflows:
+  test_app_clip:
+    steps:
+    - bitrise-run:
+        run_if: '{{enveq "BITRISEIO_STACK_ID" "osx-xcode-12.5.x"}}'
+        inputs:
+        - workflow_id: utility_test_app_clip
+        - bitrise_config_path: ./e2e/bitrise.yml
+
+  utility_test_app_clip:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/Fruta
+    - TEST_APP_BRANCH: xcode-12-gm
+    - TEST_APP_COMMIT: ""
+    - BITRISE_PROJECT_PATH: Fruta.xcodeproj
+    - BITRISE_SCHEME: Fruta iOS
+    - ARTIFACT_NAME: ios-simple-objc(1234)
+    - FORCE_CODE_SIGN_IDENTITY: "iPhone Developer: Dev Portal Bot Bitrise"
+    - FORCE_TEAM_ID: 72SA8V3WYL
+    - FORCE_PROV_PROFILE_SPECIFIER: ""
+    - IPA_EXPORT_METHOD: development
+    - IPA_EXPORT_ICLOUD_CONTAINER_ENVIRONMENT: ""
+    - XCODE_OUTPUT_TOOL: xcodebuild
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+
+  test_catalyst:
+    steps:
+    - bitrise-run:
+        run_if: '{{enveq "BITRISEIO_STACK_ID" "osx-xcode-12.5.x"}}'
```

Comment:
NOTE: we could consider to add this to the meta tag sometime

Reply:
agree, I would prefer to get back to this topic later when we have done the rest of the migrations and we see what cases we need to cover